docs/quickstart/quickstart.ipynb

```diff
@@ -2,14 +2,13 @@
  "cells": [
```

Comment:
<p>I think @wkerzendorf mentioned to show INFO level logs!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
My thinking is that we want to show the defaults in quickstart. @wkerzendorf to my memory talked about changing it so "info" is the default.